packages/atomic/src/components/search/search-box-suggestions/atomic-search-box-instant-results/atomic-search-box-instant-results.tsx

```diff
@@ -64,10 +65,6 @@ export class AtomicSearchBoxInstantResults implements InitializableComponent {
    * The maximum number of results to show.
    */
   @Prop({reflect: true}) public maxResultsPerQuery = 4;
-  /**
-   * The desired layout to use when displaying results. Layouts affect how many results to display per row and how visually distinct they are from each other.
-   */
-  @Prop({reflect: true}) public display: ResultDisplayLayout = 'list';
```

Comment:
This should not have been configurable, it's built to be list only or at least the grid isn't working properly...